OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -212,6 +212,14 @@ public AssetBrowserLogic(Widget widget, Action onExit, World world)
 				prevButton.IsVisible = () => !isVideoLoaded;
 			}
 
+			if (logicArgs.ContainsKey("SupportedFormats"))
```

Comment:
Use FieldLoader to convert to `string[]`.  This is basically reinventing the logic that already exists there.